harmonica/transformations.py

```diff
@@ -44,3 +53,276 @@ def derivative_upward(grid, order=1):
+        evenly spaced (regular grid). Its dimensions should be in the following
+        order: *northing*, *easting*. Its coordinates should be defined in the
+        same units.
+    wavelength : float
+        The cutoff wavelength in low-pass filter. Its units are the same units
+        of the ``grid``.
+
+    Returns
+    -------
+    gaussian lowpass : :class:`xarray.DataArray`
+        A :class:`xarray.DataArray` after gaussian low-pass of the passed
+        ``grid``.
+
+    References
+    ----------
+    [Geosoft1999]_
+
+    See also
+    --------
+    harmonica.filters.gaussian_lowpass_kernel
+    """
+    return apply_filter(grid, gaussian_lowpass_kernel, wavelength=wavelength)
+
+
+def gaussian_highpass(grid, wavelength):
+    """
+    Calculate the gaussian high-pass of a potential field grid
+
+    Compute the gaussian high-pass of regular gridded data using frequency
+    domain calculations through Fast Fourier Transform.
+
+    Parameters
+    ----------
+    grid : :class:`xarray.DataArray`
+        A two dimensional :class:`xarray.DataArray` whose coordinates are
+        evenly spaced (regular grid). Its dimensions should be in the following
+        order: *northing*, *easting*. Its coordinates should be defined in the
+        same units.
+    wavelength : float
+        The cutoff wavelength in high-pass filter. Its units are the same
+        units of the ``grid``.
+
+    Returns
+    -------
+    gaussian highpass : :class:`xarray.DataArray`
+        A :class:`xarray.DataArray` after gaussian high-pass of the passed
+        ``grid``.
+
+    References
+    ----------
+    [Geosoft1999]_
+
+    See also
+    --------
+    harmonica.filters.gaussian_highpass_kernel
+    """
+    return apply_filter(grid, gaussian_highpass_kernel, wavelength=wavelength)
+
+
+def reduction_to_pole(grid, i, d, im=None, dm=None):
+    """
+    Calculate the reduction to the pole of a magnetic field grid
+
+    Compute the reduction to the pole of regular gridded magnetic data using
+    frequency domain calculations through Fast Fourier Transform.
+
+    Parameters
+    ----------
+    grid : :class:`xarray.DataArray`
+        A two dimensional :class:`xarray.DataArray` whose coordinates are
+        evenly spaced (regular grid). Its dimensions should be in the following
+        order: *northing*, *easting*. Its coordinates should be defined in the
+        same units.
+    i : float in degrees
+        The inclination inducing Geomagnetic field.
+    d : float in degrees
+        The declination inducing Geomagnetic field.
+    im : float in degrees
+        The inclination of the total magnetization of the anomaly source.
+        Default is i, neglecting remanent magnetization and
+        self demagnetization.
+    dm : float in degrees
+        The declination of the total magnetization of the anomaly source.
+        Default is d, neglecting remanent magnetization and
+        self demagnetization.
```

Comment:
I'm inclined to rename this parameters to a more readable alternative like `inclination`, `declination`, `source_inclination`, `source_declination`. The same goes for the parameters of its corresponding filter.

Reply:
I rename them as `inclination`, `declination`, `magnetization_inclination`, `magnetization_declination`. I do feel it is a bit long, but it's okay..